packages/backend/src/storage/ChannelsAccessController.ts

```diff
@@ -53,7 +56,7 @@ export const createChannelAccessController = (peerId) => {
       return ''
     }
 
-    static create(_orbitdb, _options) {
+    static create(_orbitdb: OrbitDB, _type = type, _options: any) {
```

Comment:
Any possibility to replace `any` another more specyfic type in this case ?

Reply:
AccessControllers.create also has 'any' as options type so I don't think so.